.gitignore

```diff
@@ -2,5 +2,4 @@ node_modules/
 .next/
 .DS_STORE
 .env
-.env.*
-package-lock.json
```

Comment:
Let's keep `package-lock.json` in this gitignore, we don't need both this and `package.json`. Unless there is a specific package requiring a specific version number

Reply:
Ah, ok. I read online that package lock was meant to be in the repo for version control reasons, and I knew I had added it before in a prior branch so I thought that was holdover from my change and not an intentional addition. I'll get that resolved!